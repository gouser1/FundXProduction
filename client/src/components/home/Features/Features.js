import React from "react";
import { Grid, Box, Hidden, Typography } from "@material-ui/core";
import GrowthImage from "../../../images/home/growth.svg";
import ConnectImage from "../../../images/home/connect.svg";
import MobileImage from "../../../images/home/mobileview.png";
import useStyles from "./FeaturesStyle";

function Features(props) {
  const classes = useStyles();
  return (
    <div className={classes.root}>
      <Grid
        container
        alignItems="center"
        justify="center"
        style={{ minHeight: "100vh" }}
      >
        <Grid item xs={12}>
          <Box className={classes.box} textAlign="center">
            <Typography className={classes.h1}>How does it work?</Typography>
          </Box>
        </Grid>
        <Grid item lg={6} md={12} sm={12} xs={12}>
          <Box className={classes.box}>
            <Typography className={classes.h2}>
              Connecting companies to investors for maximum growth
            </Typography>
          </Box>
          <Box>
            <Typography className={classes.p1}>
              FundX makes it simple to connect companies to potential investors,
              making it easy for you to find the investment you need for your
              company to grow. Sign up today to create or Pitch or continue as a
              guest user to explore the wide range of pitches within FundX!
            </Typography>
          </Box>
        </Grid>
        <Grid item lg={6} md={12} sm={12} xs={12}>
          <Hidden mdDown>
            <Box pt={20} justifyContent="center" display="flex">
              <img src={GrowthImage} alt="" width="472px" height="451px" />
            </Box>
          </Hidden>
        </Grid>
        <Grid item lg={6} md={12} sm={12} xs={12}>
          <Hidden mdDown>
            <Box pt={20} justifyContent="center" display="flex">
              <img src={MobileImage} alt="" width="472px" height="451px" />
            </Box>
          </Hidden>
        </Grid>
        <Grid item lg={6} md={12} sm={12} xs={12}>
          <Box className={classes.box}>
            <Typography className={classes.h2}>
              Create and publish your pitch
            </Typography>
          </Box>
          <Box>
            <Typography className={classes.p1}>
              Add a pitch for your business using our create a Pitch section.
              The on-screen tips will guide you through the creation process of
              your Pitch. We've also got some great resources to help if you get
              stuck you should check our FAQ and support sections. When your
              pitch has been created, it will be listed on the site for
              prospective investors to browse and evaluate. Investors can then
              contact you if they are interested.
            </Typography>
          </Box>
        </Grid>
        <Grid item lg={6} md={12} sm={12} xs={12}>
          <Box className={classes.box}>
            <Typography className={classes.h2}>
              Explore pitches and store them for later
            </Typography>
          </Box>
          <Box>
            <Typography className={classes.p1}>
              Our main pitches section of FundX allows you to randomly search
              through the pitches to find something unique you may not have been
              looking for. You can also view all the pitches and sort through
              them to find a pitch tailored for your interest. Favouriting the
              pitches will store them in your Favourites in case you want to
              come back to it later on.
            </Typography>
          </Box>
        </Grid>
        <Grid item lg={6} md={12} sm={12} xs={12}>
          <Hidden mdDown>
            <Box pt={20} justifyContent="center" display="flex">
              <img src={ConnectImage} alt="" width="472px" height="451px" />
            </Box>
          </Hidden>
        </Grid>
      </Grid>
    </div>
  );
}

export default Features;
