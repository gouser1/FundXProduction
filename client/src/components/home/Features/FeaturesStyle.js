import { makeStyles } from "@material-ui/styles";

export default makeStyles(() => ({
  root: {
    flexGrow: 1,
    marginLeft: "5%",
    marginRight: "5%",
    marginBottom: "15%",
  },
  typography: {
    fontFamily: "'Karla', sans-serif;",
    fontSize: "1.2em",
    color: "#46637f",
  },
  h1: {
    textAlign: "center",
    fontFamily: "'Karla', sans-serif;",
    fontSize: "1.5em",
    marginBottom: "2%",
    color: "#22303d",
    fontWeight: "700",
    "@media (min-width:700px)": {
      fontSize: "2rem",
    },
  },
  h2: {
    fontFamily: "'Karla', sans-serif;",
    fontSize: "1.6em",
    fontWeight: "700",
    color: "#46637f",
    "@media (max-width:700px)": {
      fontSize: "1.3em",
      marginBottom: "4%",
      textAlign: "center",
    },
  },
  p1: {
    marginTop: "1%",
    textAlign: "left",
    fontFamily: "'Rubik', sans-serif;",
    fontSize: "1.2em",
    color: "#22303d",
    letterSpacing: "0.02em",
    lineHeight: "1.4em",
    "@media (max-width:700px)": {
      fontSize: "1.3rem",
      textAlign: "center",
    },
    "@media (max-width:1400px)": {
      fontSize: "1.1rem",
    },
  },
  box: {
    "@media (min-width:1500px)": {
      paddingTop: "8em",
    },
    "@media (max-width:1400px)": {
      paddingTop: "4em",
    },
    "@media (max-width:1000px)": {
      paddingTop: "5em",
    },
    "@media (max-width:800px)": {
      paddingTop: "4em",
    },
  },
}));
