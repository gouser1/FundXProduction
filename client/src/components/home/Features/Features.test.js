import React from "react";
import renderer from "react-test-renderer";

import Features from "./Features";

it("renders correctly", () => {
  const tree = renderer.create(<Features />).toJSON();
  expect(tree).toMatchSnapshot();
});
