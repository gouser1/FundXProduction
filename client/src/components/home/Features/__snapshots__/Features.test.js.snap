// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="makeStyles-root-1"
>
  <div
    className="MuiGrid-root MuiGrid-container MuiGrid-align-items-xs-center MuiGrid-justify-content-xs-center"
    style={
      Object {
        "minHeight": "100vh",
      }
    }
  >
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
    >
      <div
        className="MuiBox-root MuiBox-root-7 makeStyles-box-6"
      >
        <p
          className="MuiTypography-root makeStyles-h1-3 MuiTypography-body1"
        >
          How does it work?
        </p>
      </div>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-12 MuiGrid-grid-md-12 MuiGrid-grid-lg-6"
    >
      <div
        className="MuiBox-root MuiBox-root-8 makeStyles-box-6"
      >
        <p
          className="MuiTypography-root makeStyles-h2-4 MuiTypography-body1"
        >
          Connecting companies to investors for maximum growth
        </p>
      </div>
      <div
        className="MuiBox-root MuiBox-root-9"
      >
        <p
          className="MuiTypography-root makeStyles-p1-5 MuiTypography-body1"
        >
          FundX makes it simple to connect companies to potential investors, making it easy for you to find the investment you need for your company to grow. Sign up today to create or Pitch or continue as a guest user to explore the wide range of pitches within FundX!
        </p>
      </div>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-12 MuiGrid-grid-md-12 MuiGrid-grid-lg-6"
    />
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-12 MuiGrid-grid-md-12 MuiGrid-grid-lg-6"
    />
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-12 MuiGrid-grid-md-12 MuiGrid-grid-lg-6"
    >
      <div
        className="MuiBox-root MuiBox-root-10 makeStyles-box-6"
      >
        <p
          className="MuiTypography-root makeStyles-h2-4 MuiTypography-body1"
        >
          Create and publish your pitch
        </p>
      </div>
      <div
        className="MuiBox-root MuiBox-root-11"
      >
        <p
          className="MuiTypography-root makeStyles-p1-5 MuiTypography-body1"
        >
          Add a pitch for your business using our create a Pitch section. The on-screen tips will guide you through the creation process of your Pitch. We've also got some great resources to help if you get stuck you should check our FAQ and support sections. When your pitch has been created, it will be listed on the site for prospective investors to browse and evaluate. Investors can then contact you if they are interested.
        </p>
      </div>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-12 MuiGrid-grid-md-12 MuiGrid-grid-lg-6"
    >
      <div
        className="MuiBox-root MuiBox-root-12 makeStyles-box-6"
      >
        <p
          className="MuiTypography-root makeStyles-h2-4 MuiTypography-body1"
        >
          Explore pitches and store them for later
        </p>
      </div>
      <div
        className="MuiBox-root MuiBox-root-13"
      >
        <p
          className="MuiTypography-root makeStyles-p1-5 MuiTypography-body1"
        >
          Our main pitches section of FundX allows you to randomly search through the pitches to find something unique you may not have been looking for. You can also view all the pitches and sort through them to find a pitch tailored for your interest. Favouriting the pitches will store them in your Favourites in case you want to come back to it later on.
        </p>
      </div>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-12 MuiGrid-grid-md-12 MuiGrid-grid-lg-6"
    />
  </div>
</div>
`;
